### PR TITLE
CLOUDP-116808: Fix example in login command

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes
As we deprecate atlas commands in mongocli, updating the example.

Before:
```
➜  mongocli git:(master) atlasdev auth login --help
Authenticate with MongoDB Atlas.

Usage:
  atlas auth login [flags]

Examples:
  To start the interactive setup:
  $ mongocli auth login
...
```

After:
```
➜  mongocli git:(master) atlasdev auth login --help
Authenticate with MongoDB Atlas.

Usage:
  atlas auth login [flags]

Examples:
  To start the interactive setup:
  $ atlas auth login
...
```

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — COMMENTED** [review]
Could you add the ticket number and a description of the proposed changes? Thanks!

File: docs/mongocli/command/mongocli-auth-login.txt [hunk 1]
```diff
@@ -75,6 +75,6 @@ Examples
 .. code-block::
 
    To start the interactive setup:
-   $ mongocli auth login
+   $ atlas auth login
```
Comment (andreaangiolillo) [inline docs/mongocli/command/mongocli-auth-login.txt:78]: this should be mongocli as it a mongocli doc page

File: internal/cli/auth/login.go [hunk 2]
```diff
@@ -178,7 +178,7 @@ func LoginBuilder() *cobra.Command {
 		Use:   "login",
 		Short: "Authenticate with MongoDB Atlas.",
 		Example: `  To start the interactive setup:
-  $ mongocli auth login
+  $ atlas auth login
```
Comment (andreaangiolillo) [inline internal/cli/auth/login.go:181]: this doesn’t work as it adds `atlas` also to mongocli. you have to use` fmt.Sprintf(`) combined with `config.BinName()`

**Review by andreaangiolillo — CHANGES_REQUESTED** [review]

File: internal/cli/auth/login.go (hunk 2 above)
Comment (blva) [reply internal/cli/auth/login.go:181]: Thank you! Didn't think of that.

File: docs/mongocli/command/mongocli-auth-login.txt (hunk 1 above)
Comment (blva) [reply docs/mongocli/command/mongocli-auth-login.txt:78]: Fixed, thanks!

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**